src/components/footer/index.jsx

```diff
@@ -0,0 +1,35 @@
+import React from "react";
+import { makeStyles } from '@material-ui/core/styles';
+import logo from "./assets/ac_logo_24.png";
+import BottomNavigation from '@material-ui/core/BottomNavigation';
+import BottomNavigationAction from '@material-ui/core/BottomNavigationAction';
+
+
+const useStyles = makeStyles((theme) => ({
+  root: {
+    flexGrow: 1,
+    border: 0,
+    borderRadius: 3,
+    boxShadow: '0 3px 5px 2px rgba(255, 105, 135, .3)',
+    height: "10vh",
+    padding: 0,
+  },
+
+}));
+
+export default function Footer() {
+  const classes = useStyles();
+  const [value, setValue] = React.useState(0);
+
+  const handleChange = (event, newValue) => {
+    setValue(newValue);
+  };
```

Comment:
Were these being used for a test or does it have something to do with material-ui?  I ask because it doesn't look like they are being used for anything at the moment.  Better to remove them if they aren't being used.